packages/turf-isobands/index.js

```diff
@@ -0,0 +1,269 @@
+// https://github.com/RaumZeit/MarchingSquares.js
+var MarchingSquaresJS = require('./marchingsquares-isobands.min');
+
+var turfHelpers = require('@turf/helpers');
+var turfFeaturecollection = turfHelpers.featureCollection;
+var turfPolygon = turfHelpers.polygon;
+var turfMultiPolygon = turfHelpers.multiPolygon;
+var turfExplode = require('@turf/explode');
+var turfInside = require('@turf/inside');
+var turfArea = require('@turf/area');
+
+/*******************************************************************
+ * Takes a grid ({@link FeatureCollection}) of {@link Point} features with z-values and an array of
+ * value breaks and generates filled contour isobands.
+ *
+ *
+ * @module turf/isobands
+ * @category interpolation
+ * @param {FeatureCollection} grid of points, a FeatureCollection of {@link Point} features
+ * @param {string} z the property name in `points` from which z-values will be pulled
+ * @param {Array<number>} breaks where to draw contours
+ * @returns {FeatureCollection} a FeatureCollection of {@link MultiPolygon} features representing isobands
+ * @example
+ * // create random points with random
```

Comment:
Could you please elaborate a bit on this? First time I see this feature 😄 
What is missing? I just copied the JSDoc from the original version of the module